dallinger/models.py

```diff
@@ -1449,10 +1449,10 @@ class Info(Base, SharedMixin):
     #: the contents of the info. Must be stored as a String.
     contents = Column(Text(), default=None)
 
-    def __init__(self, origin, contents=None, details=None):
+    def __init__(self, origin, contents=None, details=None, failed=None):
         """Create an info."""
         # check the origin hasn't failed
-        if origin.failed:
+        if origin.failed and not failed:
             raise ValueError("{} cannot create an info as it has failed".format(origin))
```

Comment:
maybe:
raise ValueError("{} Only failed Infos can be added to this Node, as it has failed".format(origin))